helm/ph-ee-engine/values.yaml

```diff
@@ -485,6 +485,7 @@ importer_es:
   requests:
     memory: "512M"
     cpu: "100m"
+    javaToolOptions: "-Xmx1G"
```

Comment:
The default should be 256 mb as memory is 512. Heap cannot be parger than allocated memory.

Reply:
Memory defaults won't change. Xmx/heap will change.